Add docs for new sources (#11975)
PostHog/posthog.com: contents/docs/cdp/sources/temporal.md

---
title: Linking Temporal.io as a source
sidebar: Docs
showTitle: true
availability:
  free: full
  selfServe: full
  enterprise: full
---

The Temporal.io connector can link workflows and workflow history to PostHog.

To link Temporal.io:

1. Go to the [sources tab](https://us.posthog.com/pipeline/sources) of the data pipeline section in PostHog.

2. Click **+ New source** and then click **Link** next to Temporal.io.

> 3. We only allow TLS certificate connections to your temporal namespace, and so you'll need to ensure your namespace has a [certificate](https://docs.temporal.io/cloud/certificates) set up. Once done, you'll need copy all the same TLS certificate values into PostHog

Reviewer: ```suggestion
3. We only allow TLS certificate connections to your temporal namespace, so you need to ensure your namespace has a [certificate](https://docs.temporal.io/cloud/certificates) set up. Once done, copy all the same TLS certificate values into PostHog
```
